README.md

```diff
@@ -1,3 +1,78 @@
-# vtex.store-graphql
+# VTEX Store GraphQL
 
-GraphQL schema and resolvers for the VTEX api.
+## Description
+
+This project is a GraphQL API build in our [VTEX IO Platform](https://vtex.io/) as an abstraction of all VTEX public [REST API](https://help.vtex.com/developer-docs) that our commerce stores needed.
```

Comment:
I think `server` ou just `GraphQL API` is a better fit.

And also, is it an abstraction for *all* VTEX API?